src/hotspot/share/gc/shenandoah/shenandoahHeap.inline.hpp

```diff
@@ -271,7 +271,7 @@ inline void ShenandoahHeap::clear_cancelled_gc(bool clear_oom_handler) {
   _cancelled_gc.set(CANCELLABLE);
   if (_cancel_requested_time > 0) {
     double cancel_time = os::elapsedTime() - _cancel_requested_time;
-    log_info(gc)("GC cancellation took %.3fs", cancel_time);
+    log_debug(gc)("GC cancellation took %.3fs", cancel_time);
```

Comment:
may not be significant, but would prefer to avoid os::elapsedTime() invocation if we're not logging debug.

Reply:
maybe write the argument to log_debug() as os::elapsedTime() - _cancel_requested_time.

Reply:
That's a good catch and an easy change.